djangoappengine/storage.py

```diff
@@ -50,63 +73,106 @@ def serve_file(request, file, save_as, content_type, **kwargs):
     return response
```

Comment:
I'm also having trouble with the range requests here. This code looks good, but GAE seems to be serving the things incorrectly. Have you tested ranges on cloudstorage files on the dev server?